oq_wrapper/oq_wrapper/wrapper.py

```diff
@@ -251,6 +257,35 @@ def run_gmm(
         result_df = _run_oq_model(oq_model, rupture_ctx, oq_im_type_fn(), stddev_types)
 
     result_df.index = rupture_df.index
+
+    # Median prediction adjustment using sigma factor
+    # Used by 2022 NZ NSHM
+    if (
+        epistemic_branch is not constants.EpistemicBranch.CENTRAL
+        and (
+            sigma_factor_mapping := constants.GMM_EPISTEMIC_BRANCH_SIGMA_FACTOR_MAPPING.get(
+                model
+            )
+        )
+        is not None
+    ):
+        mean_cols = [col for col in result_df.columns if col.endswith("_mean")]
```

Comment:
You could do this, but I think the following is a bit more readable
```suggestion
        and model in constants.GMM_EPISTEMIC_BRANCH_SIGMA_FACTOR_MAPPING
    ):
        sigma_factor_mapping = constants.GMM_EPISTEMIC_BRANCH_SIGMA_FACTOR_MAPPING[model]
        mean_cols = [col for col in result_df.columns if col.endswith("_mean")]
```